packages/ui/exports/form-core-test-suites.js

```diff
@@ -8,3 +8,4 @@ export { runInteractionStateMixinSuite } from '../components/form-core/test-suit
 export { runNativeTextFieldMixinSuite } from '../components/form-core/test-suites/NativeTextFieldMixin.suite.js';
 export { runValidateMixinSuite } from '../components/form-core/test-suites/ValidateMixin.suite.js';
 export { runValidateMixinFeedbackPart } from '../components/form-core/test-suites/ValidateMixinFeedbackPart.suite.js';
+export { runCustomChoiceGroupMixinSuite } from '../components/form-core/test-suites/choice-group/CustomChoiceGroupMixin.suite.js';
```

Comment:
```suggestion
```
For now, we don't want to expose the mixin yet (as it probably will be merged into ChoiceGroupMixin)